### PR TITLE
Added support for message specification in deferred.reject().

### PR DESCRIPTION
**Pull request by ghost**
Also added example to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,71 @@ var SomeModel = Backbone.Model.extend({
...
+/**
+  * @param deferred Object to say whether the field is valid (call .resolve())
+  *                 or invalid (call.reject("Optional message"))
+  * @param value Value of validated attribute
+  * @param attr Name of validated attribute
+  * @param param value passed to validation definition, for example:
+  *                validation: {name: {remote: param}}
+  * @param model Object with all fields and their values
+  */
+Validation.validators.remote = function(deferred, value, attr, param, model) {
```
Comment (suevalov) [inline README.md]: @jurajMatus 
Should be Backbone.Validation.validators. Am I right?

**Comment by suevalov** [comment]
@jurajMatus 
Thanks! Few corrections:

I'd prefer less details of implementation in example. It distracts from main point.

``` javascript
/* describe function params */
Backbone.Validation.validators.customValidator = function(deferred, value, attr, param, model) {
    sendAjaxRequest(model) // some abstract function
              .success(function() {
                  deferred.resolve();
              })
              .fail(function() {
                  deferred.reject();
              });
};
```

**Comment by suevalov** [comment]
@jurajMatus After these corrections I'll merge the pull request.

**Comment by ghost** [comment]
Thanks for reading the code.
I have corrected the mistake and simplified the example.
I hope now it's better.

**Pull request merged**